pytorch_lightning/trainer/deprecated_api.py

```diff
@@ -87,3 +87,18 @@ def nb_sanity_val_steps(self, nb):
                       "`num_sanity_val_steps` since v0.5.0"
                       " and this method will be removed in v0.8.0", DeprecationWarning)
         self.num_sanity_val_steps = nb
+
+    @property
+    def fast_dev_run(self):
```

Comment:
you have mixed v0.8 and v0.9 deprecations as you can see above...
create a new deprecation mixin `TrainerDeprecatedAPITillVer0_9(ABC)`